libraries/conf_helpers.rb

```diff
@@ -48,5 +52,5 @@ def merge_config(current_config, desired_config)
       end
       stream.string
     end
-  end
+  end unless defined?(ConfHelpers)
```

Comment:
Chef still does weirdness to the objectspace by reloading these files at converge time. This is necessary for stubbing in Chefspec by ensuring the module is only defined once.